hbase-common/pom.xml

```diff
@@ -97,6 +97,15 @@
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-semconv</artifactId>
     </dependency>
+    <!-- include netty dependencies as hadoop needs them-->
```

Comment:
Is it possible to push these out to the server or test modules that make use of them? Or are we fiddling with zookeeper-server + netty classes from within hbase-common ?

Reply:
I agree i'd prefer to not have these in common if possible

Reply:
The dependencies for netty here are not about zookeeper, it is because hadoop needs them...

Reply:
Ah, shit, we only depends on hadoop-common in hbase-common, not hadoop-hdfs. Hadoop-common does not depend on netty directly but it depends on zookeeper, as while as curator...

Let me think how to only set necessary dependencies here...

Reply:
Shit I got 0e issue on my 980 Pro and my vmware disk files are corrupted...

I'm on a business trip so do not have ways to backup the files and fix the problem...

Will be back to Beijing tomorrow, please give me some moe time on this issue...

Reply:
I only include the netty4 dependency here to reflect that we depend on hadoop-common -> zookeeper, and the pre commit result seems fine. I'm not sure whether netty4 is only used by zookeeper server but it is declared as a compile scope dependency for zookeeper, so I prefer we add this dependency here.